src/dispatch/individual/service.py

```diff
@@ -44,10 +44,10 @@ def get_or_create(*, db_session, email: str, **kwargs) -> IndividualContact:
 
     if not contact:
         contact_plugin = plugins.get(INCIDENT_PLUGIN_CONTACT_SLUG)
```

Comment:
I think we need something like: `plugin_service.get_active(db_session=db_session, plugin_type="contact")`